migration/v0.37/apply/11-kube-prometheus-stack.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+
+    if [[ "${CK8S_CLUSTER}" =~ ^(sc|both)$ ]]; then
+      log_info "  - Replace kube-prometheus-stack CRDs on sc"
+      kubectl_do sc apply --server-side -f "${ROOT}"/helmfile.d/upstream/prometheus-community/kube-prometheus-stack/charts/crds/crds
+
+      log_info "  - Upgrade kube-prometheus-stack on sc"
+      helmfile_upgrade sc app=prometheus
+    fi
+    if [[ "${CK8S_CLUSTER}" =~ ^(wc|both)$ ]]; then
+      log_info "  - Replace kube-prometheus-stack CRDs on wc"
+      kubectl_do wc apply --server-side -f "${ROOT}"/helmfile.d/upstream/prometheus-community/kube-prometheus-stack/charts/crds/crds
+
+      log_info "  - Upgrade kube-prometheus-stack on wc"
+      helmfile_upgrade wc app=prometheus
+    fi
+    ;;
```

Comment:
Question: Any reason this was removed?

```suggestion
    ;;
  rollback)
    log_warn "rollback not implemented"
    ;;
  *)
    log_fatal "usage: \"${0}\" <execute|rollback>"
    ;;
```

I guess it isn't super important, but to have it be consistent it might be good to have. But I'll leave it for you to decide.